packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -953,6 +955,20 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		return columnFilterValue;
 	}
 
+	/**
+	 * Get the position of the filter box of the specified column.
+	 * @param columnName - Name of the column whose filter box position should be retrieved.
+	 * @returns The position of the filter box of the specified column.
+	 */
+	public getColumnFilterPosition(columnName: string): StarkTableColumnFilter["filterPosition"] | undefined {
+		if (!(this.filter.columns instanceof Array)) {
+			return;
```

Comment:
This should return something explicitly, in this case `undefined`

Reply:
yes that's right, I've adapted the code